src/CLClient.ts

```diff
@@ -237,6 +237,12 @@ export class CLClient {
     // Log Dialogs
     //=============================================================================
 
+    public GetLogDialogs(appId: string, packageIds: string[]): Promise<CLM.LogDialogList> {
+        const packages = packageIds.map(p => `package=${p}`).join("&")
```

Comment:
Would use one of the `querystring` modules these transformations. We should eventually remove all of them so don't want to add more.

Reply:
I'd prefer to switch them all at once

Reply:
does this work with asp.net core service that we have? will it map package=<abc>&package=<def> to packge=[abc,def]?